src/components/EpisodesNav/styles.scss

```
@@ -1,14 +1,120 @@
-.paginationLink {
-	font-size: 1em; //placeholder to have the classname showing in inspector
+.wrapper {
+	position: fixed; //with absolute it will scroll with page
+	width: 80%;
+	left: 50%;
+	bottom: 5%;
+	transform: translateX(-50%);
 }
 
-.navButton {
-	width: 50px;	//placeholder so they are visible
-	height: 50px;
-	margin: 5px;
+	.container {
+		display: flex;
```

```suggestion
		display: flex;
		position: relative;
		z-index: 1;
```
I found some z-fighting when I was testing and couldn't click the buttons.